fdbclient/include/fdbclient/SystemData.h

```diff
@@ -171,19 +195,25 @@ extern const ValueRef serverKeysTrue, serverKeysTrueEmptyRange, serverKeysFalse;
 const UID newDataMoveId(const uint64_t physicalShardId,
                         AssignEmptyRange assignEmptyRange,
                         const DataMoveType type,
+                        const DataMovementReason reason = DataMovementReason::INVALID,
```

Comment:
Do you mind removing the default value for reason? We want a reason for each data move.